avocado/utils/software_manager/backends/yum.py

```diff
@@ -31,16 +31,18 @@ class YumBackend(RpmBackend):
     Enterprise Linux.
     """
 
+    #: Path to the repository managed by Avocado
+    REPO_FILE_PATH = '/etc/yum.repos.d/avocado-managed.repo'
+
```

Comment:
Nitpick:

From pep8:

> Constants are usually defined on a **module** level and written in all capital letters with underscores separating words. 

Yes, we have the "usually", that is the reason is a "nitpick" suggestion. :) Feel free to ignore.

Reply:
BTW, the second file I looked at (Lib/smtpd.py):

```python
class SMTPChannel(asynchat.async_chat):
    COMMAND = 0
    DATA = 1

    command_size_limit = 512
    command_size_limits = collections.defaultdict(lambda x=command_size_limit: x)
```

Two styles right next to each other.